src/emulator/cartridges/CartridgeMbc1.ts

```diff
@@ -78,10 +78,10 @@ export class CartridgeMbc1 extends CartridgeBase {
         const ramBanks = Math.max((ramSize / 8) | 0, 1);
 
         const romSlices = new Array(romBanks);
-        for (let i = 0; i < romBanks; i++) romSlices[i] = this.image.subarray(i * 0x4000, (i + 1) * 0x4000);
+        for (let i = 0; i < romBanks; i++) romSlices[i] = this.image.subarray(i * CartridgeROMBankSize, (i + 1) * CartridgeROMBankSize);
```

Comment:
Hmhm, ich bin in dem Kontext ja eher ein Freund der magic numbers. Die gehören halt zu der Hardware dazu wie der Senf zur Weisswurscht und werden sich auch niemals ändern --- da finde ich persönlich die nackte Zahl irgendwie ehrlicher, eine Konstante suggeriert mir irgendwo, dass die von irgendwas abhängt und sich auch mal ändern könnte, und die Zahl ist auch herzlich einfach.

Das ist aber Geschmackssache, und auf die Schienen lege ich mich deswegen aber nicht --- aber in den Raum stellen will ich's zumindest mal 😏